osu.Game/Overlays/OnScreenDisplay.cs

```diff
@@ -162,56 +91,23 @@ public void StopTracking(object source, ITrackableConfigManager configManager)
                 return;
 
             existing.Unload();
-            existing.SettingChanged -= display;
+            existing.SettingChanged -= displayTrackedSettingChange;
 
             trackedConfigManagers.Remove((source, configManager));
         }
 
-        private void display(SettingDescription description)
+        /// <summary>
+        /// Displays the given <see cref="Toast"/> as parameter on the OSD
```

Comment:
```suggestion
        /// Displays the provided <see cref="Toast"/> temporarily.
```

Reply:
why not just make `DisplayTemporarily` public and use that? this is the only call site of that function.

Reply:
DisplayTemporarily contains mostly transformation code and is actually overriden in tests to show the toast without the fading out

![image](https://user-images.githubusercontent.com/20256717/62854255-10687e80-bcef-11e9-8db2-12d304ea08fa.png)